content/docs/rfcs/72/README.md

```diff
@@ -0,0 +1,273 @@
+---
+slug: 72
+title: 72/WAKU-RLN-KEYSTORE
+name: Waku RLN Keystore
+status: raw
+category: Standards Track
+editor: Jimmy Debe <jimmy@status.im>
+contributors: 
+- ?
+---
+
+# Abstract
+Encrypted credentials are stored in a JSON schema to securely exchange credentials between peers.
+
+# Summary
+A keystore is a construct to store a user’s keys. 
+The keys will be encrypted and decrypted based on methods specified in the specification. 
+This keystore specification uses [32/RLN-V1](https://rfc.vac.dev/spec/32/), Rate Limit Nullifiers, as a spam-prevention mechanism by generating zero-knowledge proofs and storing the proofs locally in the keystore.
```

Comment:
proofs are not stored in the keystore
Rate Limiting Nullifier, btw!

Reply:
should I use RLNV1 or V2 in this spec? @rymnc

Reply:
rln-v1 for now, thanks